src/main/java/uk/nhs/hee/tis/trainee/sync/mapper/ReferenceMapper.java

```diff
@@ -52,16 +52,19 @@ public interface ReferenceMapper {
   ReferenceDto toReference(Record recrd);
 
   /**
-   * The LocalOfficeContact table does not have a status field, so set this manually.
+   * The LocalOfficeContact table does not have a status field, so set this manually. The tisId
+   * field is also missing: there is only a UUID-style id field.
    *
    * @param recrd  The record source.
    * @param target The DTO target.
    */
   @AfterMapping
-  default void patchLocalOfficeContactStatus(Record recrd, @MappingTarget ReferenceDto target) {
+  default void patchLocalOfficeContactAndType(Record recrd, @MappingTarget ReferenceDto target) {
     if (recrd.getTable() != null
-        && recrd.getTable().equalsIgnoreCase("LocalOfficeContact")) {
+        && (recrd.getTable().equalsIgnoreCase("LocalOfficeContact")
+        || recrd.getTable().equalsIgnoreCase("LocalOfficeContactType"))) {
       target.setStatus(CURRENT);
+      target.setTisId(recrd.getData().get("id"));
```

Comment:
This is a bit of a hack, but it saves a lot of de-standardisation in the downstream reference service. Given the new LocalOffice[x] tables look different to the existing tables, this felt like the least intrusive place to tweak them to fit.